packages/components-library/src/form-field.tsx

```diff
@@ -35,7 +35,8 @@ export interface FormFieldProps {
         | "text"
         | "url"
         | "datetime-local"
-        | "date";
+        | "date"
+        | "password";
```

Comment:
Why have we included this `password` property here?

Reply:
Its for just remove typescript error.